system-addon/content-src/lib/constants.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   // constant to know if the page is about:newtab or about:home
-  IS_NEWTAB: document.documentURI === "about:newtab"
+  IS_NEWTAB: global.document && global.document.documentURI === "about:newtab"
```

Comment:
What is it that keeps this code running in the browser?  Webpack magic?  Do we assign global=Window somewhere?

Reply:
`global` is `window` in webpack, yes